### DISCUSSION TITLE
How to suspend the RF interface of the eNodeB / BladeRF

### DISCUSSION THREAD

**Discussion by mullerch** [srsran/srsRAN_4G#1351] [Q&A]
Hi,

Is there a way to temporarily suspend the RF interface of the eNB so that it does not emit ? Either a command for srsRAN or directly interacting with the BladeRF ? With the possibility to resume a minute later without long interruption. If I shutdown the eNB software, it can take up to a minute for it to be ready again.

Context : we have srsRAN setup for testing purpose of a LTE device. These tests are done in a RF chamber to avoid any issues with telecom regulation. The test chamber is frequently opened/closed to change the device. I would like to interrupt the RF interface at that time.

**Comment by pgawlowicz** [comment]
You can set the cell gain to a very small number using srsgnb console:
https://github.com/srsran/srsRAN_4G/blob/master/srsenb/src/main.cc#L541

**Reply by mullerch** [reply]
Thanks for the answer, I'll give it a try.
